src/main/java/backend/resource/TurboMilestone.java

```diff
@@ -83,6 +86,60 @@ public TurboMilestone(TurboMilestone milestone) {
     private void ______METHODS______() {
     }
 
+    public boolean isOverdue() {
+        return dueDate.isPresent() && dueDate.get().isBefore(LocalDate.now());
+    }
+
+    public boolean hasOpenIssues() {
+        return openIssues > 0;
+    }
+
+    /**
+     * A milestone is ongoing if it is open and not due yet or if it is overdue but still open and has open issues.
+     */
+    public boolean isOngoing() {
+        return isOpen() && (!isOverdue() || hasOpenIssues());
+    }
+
+    public static List<TurboMilestone> filterMilestonesOfRepos(List<TurboMilestone> milestones,
+                                                               List<String> repoIds) {
+        return milestones.stream()
+                .filter(ms -> repoIds.contains(ms.getRepoId().toLowerCase()))
+                .collect(Collectors.toList());
+    }
+
+    public static List<TurboMilestone> getOpenMilestones(List<TurboMilestone> milestones) {
```

Comment:
...I feel that `filter` sounds weird because filter methods usually require some kind of "filter condition" as its parameter. What do you think?